### DISCUSSION TITLE
Created mutlibody disappears upon simulation

### DISCUSSION THREAD

**Discussion by akhilsathuluri** [bulletphysics/bullet3#3893] [Q&A]
I create a new instance of my robot which is loaded using `loadURDF`. The new instance is created using `createMultiBody`. Upon `stepSimulation`, the newly created multibody disappears and the query for it using `findBasePositionAndOrientation` returns `((nan, nan, nan), (nan, nan, nan, nan))`. However the initially loaded original model behaves as expected.
Can someone please help me with whats going on here?

**Comment by erwincoumans** [comment]
Perhaps the initial state is overlapping or the URDF is not valid.
